docs/knowledgebase/advanced/account-info.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Account Information
+---
+
+## `Account` StorageMap
+
+In Substrate, account information is stored in a
+[**Storage Map**](https://substrate.dev/rustdocs/v3.0.0/frame_support/storage/types/struct.StorageMap.html#impl),
+extracted and shown below:
+
+[`frame/system/src/lib.rs`](https://github.com/paritytech/substrate/blob/bcd649ffca9efc93f8b4ac1506ec8117b71e1aac/frame/system/src/lib.rs#L530-L538):
+
+```rust
+/// The full account information for a particular account ID.
+#[pallet::storage]
+#[pallet::getter(fn account)]
+pub type Account<T: Config> = StorageMap<
+  _,
+  Blake2_128Concat,
+  T::AccountId,
+  AccountInfo<T::Index, T::AccountData>,
+  ValueQuery,
+>;
+```
+
+`Account` storage map takes in five type parameters, so it has no prefix, use `Blake2_128Concat` as
+the hash algorithm, use `T::AccountId`, the account public key as the map key, with
+`AccountInfo<T::Index, T::AccountData>` as the map value.
```

Comment:
its really strange for you to say all this again when posting the struct above, idk, maybe just me.

If you want to explain the Account storage item, a simple explaination like:

"The account storage map simply maps each account id on the blockchain to an `AccountInfo` struct."